backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java

```diff
@@ -296,11 +296,6 @@ public String getOrderingProviderID() {
     return provider.map(Provider::getProviderId).orElse(null);
   }
 
-  @JsonProperty("First_test")
```

Comment:
Have we confirmed this change with the RS team?

Reply:
who should I reach to verify?

Reply:
I pinged the rs-eng channel about this – I think for now we should send null or UNK, as this field isn't required per HHS, but it always feels a little risky to change the columns we send